### ISSUE TITLE
[wg/webrtc] Web Real-Time Communications Working Group Charter

### ISSUE THREAD

**Issue by caribouW3** [w3c/strategy#563]
# Charter Review

[Charter](https://w3c.github.io/charter-drafts/2026/webrtc-wg.html)

[diff from charter template](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https://w3c.github.io/charter-drafts/2026/webrtc-wg.html)

Expected end of charter refinement phase: 2026-08-25

If applicable:

[diff from previous charter](https://services.w3.org/htmldiff?doc1=https://www.w3.org/2024/08/webrtc-charter.html&doc2=https://w3c.github.io/charter-drafts/2026/webrtc-wg.html)

[chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/webrtc)

What kind of charter is this? 
* Existing 

**Also add a "card" for this issue to the [Strategy Funnel](https://github.com/orgs/w3c/projects/97/views/2).**

## Substantive changes

No substantive change in scope.
A small number of drafts will be retired as their content is mature enough to be moved to specifications that are more advanced on the Rec track. Some specs in CR expected to reach REC within this charter timeframe.

## Where would charter proponents like to see issues raised?

Discussion in this issue. Charter amendment proposals in the [charter-drafts](https://github.com/w3c/charter-drafts/issues?q=is%3Aissue%20state%3Aopen) GH repository.

Note: proposed chairs should be copied @... on this issue.

@youennf @jan-ivar @guidou

**Comment by tidoust** [comment]
OK to initiate charter refinement!

**Comment by plehegar** [comment]
[Charter refinement](https://lists.w3.org/Archives/Member/w3c-ac-members/2026JulSep/0021.html)

**Comment by simoneonofri** [comment]
Hello, I have two security comments/questions:

1. [SFrame is presented as its replacement](https://w3c.github.io/charter-drafts/2026/webrtc-wg.html#webrtc-identity), but the security properties are not equivalent, could the charter name the successor deliverable for the identity, target-peer, and media-isolation properties, or explicitly re-scope them, before discontinuation?
   - WebRTC Identity covers peer identity and isolated media
   - SFrame protects encoded frames, leaves key management to the application, and [does not provide per-sender authentication](https://www.rfc-editor.org/rfc/rfc9605.html#section-7.2).

2. Compared with the previous charter, this one removes the concept of preserving user control and revocation over local-device access. Was this removal intentional? If not, could the charter restore scoped commitments?

**Comment by jan-ivar** [comment]
Hi Simone, SFrame and WebRTC Identity are unrelated technologies with different histories, so one does not _"replace"_ the other, as you correctly point out. So that's perhaps unfortunate phrasing in the proposed plan language. Thanks for catching it!

> ...could the charter name the successor deliverable for the identity, target-peer, and media-isolation properties, or explicitly re-scope them, before discontinuation?

WebRTC-Identity is being deprecated, not because it's been replaced, but for lack of implementer interest (no substantial updates since 2018). As such there is no successor deliverable for those things, at least not in this WG.

But I think SFrame in conjunction with some [future MLS API](https://github.com/mozilla/explainers/blob/main/MessagingLayerSecurity.md) might someday solve some (but not all) of the same use cases. So I'm the one who asked the WG to hold off on deprecating WebRTC-Identity until SFrame has two implementations. I suspect that's the source of the conflation.

> Compared with the previous charter, this one removes the concept of preserving user control and revocation over local-device access. Was this removal intentional? 

Yeah this looks like a mistake to me too. @caribouW3 wdyt?

**Comment by youennf** [comment]
Wrt to item 1, the charter proposal currently states:
`Secure Frame (SFrame) added in WebRTC extension should replace this, covering data in addition to audio and video, so this document will be discontinued.`

We could replace it with something like:
`This document will be discontinued due to lack of interest. Secure Frame (SFrame) added in WebRTC Encoded Transform  will cover a limited set of use cases mentioned in WebRTC-Identity.`

**Comment by matatk** [comment]
No concerns from APA.

**Comment by himorin** [comment]
No comment or request from i18n
